### ISSUE TITLE
@import EarlGrey not working

### ISSUE THREAD

**Issue by AshishGogna** [google/EarlGrey#27]
I followed all the steps mentioned here - https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md.
I followed Github installation, but when i  tried to import earlgrey using this `@import EarlGrey`, it shows me an error saying _"Module 'EarlGrey' not found"_.

I followed those steps exactly as they are mentioned.

**Comment by tirodkar** [comment]
@AshishGogna, to repro the issue from our side, is this a Swift project or an Objective-C project? In case of a Swift project, have you also added [EarlGrey.swift](https://github.com/google/EarlGrey/blob/master/Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGrey.swift) to your project?

Does `#import <EarlGrey/EarlGrey.h>` work instead of `@import EarlGrey;`?

**Comment by AshishGogna** [comment]
I tried `#import <EarlGrey/EarlGrey.h>`.

Did a clean integration (followed all the steps again, used another directory) and it worked.

**Comment by brunomunizaf** [comment]
Just wanted to add that i had the same issue and what was causing this was that on the podfile i was not using it on the proper target. For example, my target name was 'Tests' so i should use:
**target 'Tests' do
pod 'EarlGrey'
end**
Instead of adding the pod to the **target 'AppName' do ...**

**Comment by IcyW** [comment]
> I followed all the steps mentioned here - https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md.
> I followed Github installation, but when i tried to import earlgrey using this `@import EarlGrey`, it shows me an error saying _"Module 'EarlGrey' not found"_.
> 
> I followed those steps exactly as they are mentioned.

Solved this problem by adding `EarlGrey.framework` in  「EarlGreyExampleTests-Build Phases-Link Binary With Libraries」...  And I found the path of `EarlGrey.framework` from  「EarlGreyExampleSwiftTests-Build Phases-Link Binary With Libraries」 , where  `EarlGrey.framework` was already added. 
![image](https://user-images.githubusercontent.com/16513834/88901314-3c27a000-d283-11ea-9eb5-9a7e81d71b0d.png)
![image](https://user-images.githubusercontent.com/16513834/88901428-58c3d800-d283-11ea-80b5-c3bc23321e17.png)

Not sure if it is the source of the issue.
In case other people need it.
